scripts/utils.js

```diff
@@ -119,7 +119,7 @@ export function getPluralForm(number, forms) {
 
 
 export async function openBackgroundTab(url) {
-    let tab = await browser.tabs.getCurrent();
+    let [tab] = await browser.tabs.query({active: true, currentWindow: true});
```

Comment:
This seems to be not a good idea. It'll cause the new tab to be opened as if the currently active tab opened it, but the currently active tab is just the tab you happened to be on when you peeked into Brief... does it really make sense to bind the new tab to it?

Reply:
I figured this was a bug, since it seems the original intention was to achieve the same goal using `browser.tabs.getCurrent()` (which does not return the current tab when called from a background script).
This fix makes it so that when Brief is closed the last tab you were on is displayed, which is usually the tab I want (returning to the workflow after a quick peek). This is also the purpose of the popup feature.